cmd/app/start/start_test.go

```diff
@@ -360,14 +355,26 @@ func TestStartCommand_Execute(t *testing.T) {
 			requirementsController: requirementsMock,
 		}
 
+		oldStdout := os.Stdout
+
+		r, w, _ := os.Pipe()
+		os.Stdout = w
+		outC := make(chan string)
+		go func() {
+			var buf bytes.Buffer
+			io.Copy(&buf, r)
+			outC <- buf.String()
+		}()
 		cobraCmd := cmd.CreateStartCommand()
-		cobraCmd.SetOut(stdoutMock)
+		cobraCmd.SetOut(w)
 		cobraCmd.SetArgs([]string{"-p", "./", "-o", "json", "-O", "./tmp-json.json"})
 
 		assert.NoError(t, cobraCmd.Execute())
-		outputBytes, err := ioutil.ReadAll(stdoutMock)
-		output := string(outputBytes)
+		err := w.Close()
 		assert.NoError(t, err)
+		os.Stdout = oldStdout
+		output := <-outC
+
```

Comment:
This seems to be a hack to me, I think that we can change the stdout of cli without changing the global `os.Stdout` value.